server/service/src/sync/test/integration/remote/requisition.rs

```diff
@@ -116,7 +116,7 @@ impl SyncRecordTester for RequisitionRecordTester {
             d.suggested_quantity = 15;
             d.supply_quantity = 15;
             d.available_stock_on_hand = 15;
-            d.average_monthly_consumption = 20;
+            d.average_monthly_consumption = 10;
```

Comment:
changed here to make sure devision by number_of_days_in_a_months is a recursive decimal